engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -35,9 +39,11 @@
  * TODO: Add diagram of this node
  */
 public class AmbientOcclusionPassesNode extends AbstractNode {
+    public static final ResourceUrn SSAO_URN = new ResourceUrn("engine:ssao");
```

Comment:
I think I mentioned would be good to remove all the _URN suffixes from variables names. With such a simple type feels a bit like writing `int counter_int;`